WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -213,14 +214,26 @@ public void onClick(View v) {
                                         }
                                     }).show();
                         } else if (savedLocally) {
-                            Snackbar.make(getActivity().findViewById(R.id.coordinator),
-                                    R.string.editor_post_saved_locally_not_published, Snackbar.LENGTH_LONG)
-                                    .setAction(R.string.button_publish, new View.OnClickListener() {
-                                        @Override
-                                        public void onClick(View v) {
-                                            publishPost(post);
-                                        }
-                                    }).show();
+                            if (isScheduledPost) {
```

Comment:
Btw, just noting here that the code example you have provided won't work, since those are not strings and just resources :)